src/app/__init__.py

```diff
@@ -0,0 +1,47 @@
+from flask import Flask
+from werkzeug.routing import BaseConverter
+from werkzeug.http import HTTP_STATUS_CODES
+from .helpers import get_view_args, chapter_lang_exists, ebook_exists, get_ebook_methodology, \
+    add_footnote_links, year_live, accentless_sort
+from config import TEMPLATES_DIR, STATIC_DIR
+import logging
+
+
+# Set WOFF and WOFF2 caching to return 1 year as they should never change
+# Note this requires similar set up in app.yaml for Google App Engine
+class MyFlask(Flask):
```

Comment:
Something more project-specific

```suggestion
class WebAlmanacServer(Flask):
```